### PR TITLE
adding monthly/yearly leo skus for prod/staging

### PR DESCRIPTION
**Pull request by husobee**
### Summary

adding monthly/yearly leo skus for prod/staging

### Type of change ( select one )

- [ ] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [ ] Other

### Tested Environments

- [ ] Development
- [ ] Staging
- [ ] Production

### Before submitting this PR:

- [ ] Does your code build cleanly without any errors or warnings?
- [ ] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you squashed all intermediate commits?
- [ ] Is there a clear title that explains what the PR does?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security / privacy review if needed
- [ ] Have you performed a self review of this PR?

### CODE REVIEW

File: tools/macaroon/cmd/brave-leo/premium_prod_time_limited_v2.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+tokens:
+  - id: "brave-leo-premium"
+    version: 1
+    location: "leo.brave.com"
```
Comment (nvonpentz) [inline tools/macaroon/cmd/brave-leo/premium_prod_time_limited_v2.yaml:4]: What's the function of `location` - does this need to map to a domain that we're actually using? Right now aichat servers are hosted at aichat.bsg.brave.com, but I'm not sure if that's even relevant.

File: tools/macaroon/cmd/brave-leo/premium_prod_time_limited_v2.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    first_party_caveats:
+      - sku: "brave-leo-premium"
+      - price: 15.00
+      - currency: "USD"
+      - description: "brave-leo-premium"
+      - credential_type: "time-limited-v2"
+      - credential_valid_duration: "P1M"
+      - each_credential_valid_duration: "P1D"
+      - expires_after: "P1M"
```
Comment (nvonpentz) [inline tools/macaroon/cmd/brave-leo/premium_prod_time_limited_v2.yaml:13]: What is the difference between these three:
```
      - credential_valid_duration: "P1M"
      - each_credential_valid_duration: "P1D"
      - expires_after: "P1M"
```

File: tools/macaroon/cmd/brave-leo/premium_prod_yearly_time_limited_v2.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      - issuer_token_overlap: "0"
+      - allowed_payment_methods: "stripe"
+      - metadata: '
+{
+  "stripe_product_id": "prod_O9uKDYsRPXNgfB",
+  "stripe_item_id": "price_1NXmfTBSm1mtrN9nybnyolId",
+  "stripe_success_uri": "https://account.brave.com/account/?intent=provision",
+  "stripe_cancel_uri": "https://account.brave.com/plans/?intent=checkout"
+}'
+
```
Comment (nvonpentz) [inline tools/macaroon/cmd/brave-leo/premium_prod_yearly_time_limited_v2.yaml:24]: Why do we have both monthly and yearly configs?

**Review by nvonpentz — APPROVED** [review]

**Comment by nvonpentz** [comment]
I'm fine just using `accounts.bravesoftware.com` to develop against on the browser side unless there are considerations I'm not aware of

**Review by pavelbrm — APPROVED** [review]
👍

**Pull request merged**